resources/views/createCourse.blade.php

```diff
@@ -0,0 +1,154 @@
+
+@extends('layouts.app')
+
+@section('title', 'Register a course')
+
+@section('content')
+<div class="padded content">
+	<h1>Register a course</h1>
+	<form class="ui error form" method="POST" action="/courses">
+		@csrf
+		<!-- Course name -->
+		<div class="field">
+			<label for="courseName">Course name</label>
+			<input type="text" name="courseName" id="courseName" required>
+		</div>
+		<!-- Teacher(s) -->
+		<div class="field" title="Are you giving this course with another teacher?">
+			<label for="teachers">Teacher(s)</label>
+			<select class="ui fluid search dropdown" name="teachers" id="teachers" multiple required>
```

Comment:
Add "[]" at the end of the name attribute, to ensure this input's contents are sent as a list to back-end

Reply:
Done, arigatou!